pkg/private/deb/make_deb.py

```diff
@@ -348,6 +348,10 @@ def main():
   parser.add_argument(
       '--triggers',
       help='The triggers file (prefix with @ to provide a path).')
+  parser.add_argument(
+      '--stamp',
+      action='append',
```

Comment:
Why is this not a bool?  Can you update the description do describe what feature is being added?